package.json

```diff
@@ -11,7 +11,7 @@
     "test": "NODE_ICU_DATA=node_modules/full-icu jest --maxWorkers=3 --coverage=false",
     "check": "lerna run check",
     "release": "lerna publish",
-    "clean": "rimraf './packages/*/dist'",
+    "clean": "rimraf './packages/*/dist' './packages/polyfills/tsconfig.tsbuildinfo'",
```

Comment:
Kinda icky but couldn't think of quicker options. The _more correct_ way to do this would be to add a `tsconfig.json#tsBuildInfoFile` path to every package. I don't have the bandwidth to do that right now, this is a _hot fix_-ey kinda PR to unblock consumers.

Reply:
Why does this fix the problem?

Reply:
Explicitly specifying the `tsconfig.json#tsBuildInfoFile` doesn't fix anything per-se. It just prevents the need for cleaning this lingering  `tsconfig.tsbuildinfo` since we explicitly specify where we want this file to go.